cdk-infra/lib/constructs/api/webhook-env-construct.ts

```diff
@@ -21,7 +21,7 @@ export class WebhookEnvConstruct extends Construct {
     const featureName = getFeatureName();
 
     // Create configurable feature flag that lives in parameter store.
-    const monorepoPathFeature = new StringParameter(this, 'monorepoPathFeature', {
+    const featureFlagMonorepoPath = new StringParameter(this, 'monorepoPathFeature', {
       parameterName: `${ssmPrefix}/${featureName}/monorepo/path_feature`,
       stringValue: env === 'dotcomstg' || env === 'stg' ? 'true' : 'false',
     });
```

Comment:
Is the Param Store value for `monorepo/path_feature` different from the one for the new `flag/monorepo_path`, or are these intended to be the same? If they're intended to be the same, we should probably stick to one (maybe `flag/monorepo_path` for consistency with our other feature flags?)

Reply:
True! This was the one added by @branberry for the Enhanced Autobuilder. Need to check with him to see how feasible and easy changing it would be. It seems to be dynamic, so probably just here... but I want to keep backwards compatibility if it's in use